prepare_setup.sh

```diff
@@ -4,7 +4,7 @@ go_version=""
 
 go_dir=/usr/local
 
-helm_version="v3.4.0"
+helm_version="v3.5.5"
```

Comment:
I tried to run the script, but it got 404 error, 
i think the way the scripts fetch the binary should change to follow https://helm.sh/docs/intro/install/#from-script. i will do it and add it to this PR if you would like?

Reply:
It's a type. It should be v3.5.3

Reply:
> I think the way the scripts fetch the binary should change to follow https://helm.sh/docs/intro/install/#from-script. I will do it and add it to this PR if you would like?
@AbdYsn I prefer to use some the same version on the CI for the stability and making builds reproducible